Snakefile

```diff
@@ -87,6 +102,24 @@ rule create_network:
         "scripts/create_network.py"
 
 
+rule clean_earth_osm_data:
+    input:
+        #buildings_json="resources/buildings/buildings.json",
+        microgrid_shapes="resources/shapes/microgrid_shapes.geojson",
+    output:
+        buildings_geojson="resources/buildings/buildings.geojson",
+        microgrids_buildings="resources/buildings/microgrids_buildings.geojson",
+    log:
+        "logs/clean_earth_osm_data.log",
+    benchmark:
+        "benchmarks/clean_earth_osm_data"
+    threads: 1
+    resources:
+        mem_mb=3000,
+    script:
+        "scripts/clean_earth_osm_data.py"
+
+
 rule build_renewable_profiles:
```

Comment:
the regions in build_renewable_profiles had to be adapted to the microgrids, not sure where it got lost.
I tried to use the microgrid shapes but the format does not match. What is the shape to use?

I also found two microgrid_shapes and _bus_shapes in resources, what is the difference?